src/components/dashboard/main-content/course-enrollments/course-cards/AssignedCourseCard.jsx

```diff
@@ -39,6 +44,7 @@ const AssignedCourseCard = (props) => {
       type={COURSE_STATUSES.assigned}
       hasViewCertificateLink={false}
       canUnenroll={false}
+      externalCourseLink={false}
```

Comment:
[inform] Extended the `BaseCourseCard` to use React Router's `Link` if the `linkToCourse` prop is meant to be directing users to an internal page route within the same application.